scripts/codestyle/run_uncrustify.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+if [ $# -gt 3 ]; then
+	echo "Too many parameters specified"
+	exit 1
+fi
+
+VERBOSE=false
+HASHES=()
+
+while [[ $# -gt 0 ]]
+do
+	key="$1"
+	case $key in
+	    -v|--verbose)
+	    VERBOSE=true
+	    ;;
+	    *)
+	    HASHES+=($key)
+	    ;;
+	esac
+	shift
+done
+
+if [ ${#HASHES[@]} -eq 2 ];
```

Comment:
Could you specifiy usage in help message or comment.
Is one hash mode supported (semantic: from that commit to current)
Two hash mode is doubtfully needed from my point, are you find it useful?